awxkit/awxkit/api/pages/organizations.py

```diff
@@ -43,6 +43,18 @@ def remove_galaxy_credential(self, credential):
     def payload(self, **kwargs):
         payload = PseudoNamespace(name=kwargs.get('name') or 'Organization - {}'.format(random_title()),
                                   description=kwargs.get('description') or random_title(10))
+        
+        for fk_field in ('inventory', 'default_environment'):
```

Comment:
why do you have this `inventory` item in this list? I think you need to remove that